design-system-docs/package.json

```diff
@@ -8,19 +8,19 @@
     "update-colour-data": "npx sass-export --dependencies='../scss/1-settings/' ../scss/1-settings/_colour-language.scss -o src/_data/colour_language.json"
   },
   "devDependencies": {
-    "css-loader": "~6.7.4",
-    "esbuild": "~0.16.16",
-    "esbuild-loader": "~2.20.0",
+    "css-loader": "~6.8.1",
+    "esbuild": "~0.18.2",
+    "esbuild-loader": "~3.0.1",
     "file-loader": "^6.2.0",
     "mini-css-extract-plugin": "^1.3.1",
```

Comment:
There was a new release of `mini-css-extract-plugin`, however it contains a breaking change with the `publicPath`: https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md#-breaking-changes